### ISSUE TITLE
Returning a Promise

### ISSUE THREAD

**Issue by JSteunou** [bendc/animateplus#19]
Hi,

first thank you first this jewel, really a saviour! 

Could you consider returning a Promise from `animate` call?

Would solve https://github.com/bendc/animateplus/issues/6 and allow use of future async / await out of the box.

**Comment by jhabdas** [comment]
@JSteunou would you entertain a pull in this regard?

**Comment by jhabdas** [comment]
@JSteunou redacted

**Comment by jhabdas** [comment]
Agreed. The amount of spaghetti code I wrote trying to do a simple headline rotation has lead me to giving [animo](https://github.com/ThrivingKings/animo) a shot instead (it returns a promise, but uses a different approach)

**Comment by msheakoski** [comment]
What about using a simple wrapper?
```javascript
function pranimate(params) {
  return new Promise(function (resolve, reject) {
    params.complete = resolve;
    animate(params);
  });
}
```
```html
<h1>Hello</h1>
<script> 
  document.addEventListener("DOMContentLoaded", function(event) {    
    pranimate({
      el: "h1",
      translateX: [0, 200],
      duration: 1000
    }).then(function () {
      return pranimate({
        el: "h1",
	translateX: [200, 200],
        translateY: [0, 200],
        duration: 1000
      });
    }).then(function () {
      return pranimate({
        el: "h1",
	translateX: [200, 0],
        translateY: [200, 200],
        duration: 1000
      });
    }).then(function () {
      return pranimate({
        el: "h1",
        translateY: [200, 0],
        duration: 1000
      });
    });
  });
</script>
```

**Comment by bendc** [comment]
Fixed in v2.0.0
